engines/download_only.py

```diff
@@ -124,11 +136,11 @@ def dummy_method(self, *args, **kwargs):
                 }
 remove_methods = ['insert_data_from_file']
 for name, method in methods:
-    if (not name in keep_methods
-        and not 'download' in name
-        and not 'file' in name
-        and not 'dir' in name):
-
+    if (
+            name not in keep_methods and
+            'download' not in name and
+            'file' not in name and
+            'dir' not in name):
```

Comment:
It seems like Pep 8, such as it provides guidance in this case, recommends keeping the first condition with the `if (`:
https://www.python.org/dev/peps/pep-0008/#indentation